cloud/db_cloud_impl.cc

```diff
@@ -405,6 +406,23 @@ Status DBCloudImpl::DoCheckpointToCloud(
     return st;
   }
 
+  // Upload MANIFEST and CLOUDMANIFEST sequentially only after copying all data
+  // files
+
+  // MANIFEST file
+  st = upload_file(cenv->GetStorageProvider(), tmp_manifest_fname,
+                   manifest_fname);
+  if (!st.ok()) {
+    return st;
+  }
+
+  // CLOUDMANIFEST file
+  st = upload_file(cenv->GetStorageProvider(), cenv->CloudManifestFile(""),
```

Comment:
Curious when would this `DoCheckPointToCloud` function be called in our system? Just realized we might need to change this for leader/follower.

Reply:
Right now it should only be called by bulk leaves